qt/scientific_interfaces/EnggDiffraction/IEnggDiffGSASFittingObserver.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MANTIDQTCUSTOMINTERFACES_ENGGDIFFRACTION_GSASFITTINGOBSERVER_H_
+#define MANTIDQTCUSTOMINTERFACES_ENGGDIFFRACTION_GSASFITTINGOBSERVER_H_
+
+#include "GSASIIRefineFitPeaksOutputProperties.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+class IEnggDiffGSASFittingObserver {
+
+public:
```

Comment:
This class has `virtual` methods. How about a `virtual` destructor as well? Could be declared `default`.